src/libstd/lib.rs

```diff
@@ -454,6 +454,7 @@ pub mod f64;
 pub mod thread;
 pub mod ascii;
 pub mod backtrace;
+pub mod bench;
```

Comment:
Shouldn't the module also have a stability annotation?

Reply:
Yes? It’s near the top of `libstd/bench.rs`, like for most other modules under `std::`

Reply:
I saw that, but I guess it isn't necessary on the mod item itself?

Reply:
This (inner) attribute is already on the mod item.  If we switch to inline module syntax, this PR is equivalent to:

```rust
pub mod bench {
    #![unstable(…)]
    // …
}
```

Which is equivalent to:


```rust
#[unstable(…)]
pub mod bench {
    // …
}
```